### PR TITLE
Addconfig

### PR DESCRIPTION
**Pull request by smileandyxu**

### CODE REVIEW

**Comment by smileandyxu** [comment]
Finished: 
View: all configurations, including learning rate, loss function, activatation function; add tutorial and about.
ViewModel&Model: provides attribute interfaces.

All Clear!
